app/http.php

```diff
@@ -634,6 +634,11 @@ function (string $runtimeId, string $payload, array $variables, int $timeout, st
                 }
             }
 
+            // Update swoole table
+            $runtime = $activeRuntimes->get($activeRuntimeId);
```

Comment:
This should be done before the createRuntimeRequest.

The runtime server can still get deleted by the maintenance worker

Reply:
Moved above createRuntimeRequest